packages/react-vapor/src/components/slider/examples/SliderExamples.tsx

```diff
@@ -2,6 +2,9 @@ import * as React from 'react';
 import {Slider} from '../Slider';
 
 export class SliderExamples extends React.Component<any, any> {
+    static description =
+        'Sliders are used to adjust contiguous values (e.g., screen brightness settings) that do not require a high level of precision or a range of related values (e.g., screen resolution settings).';

```

Comment:
Do you mean 
"...that do not require (a high level of precision), nor (a range of related values)..."
or "...that do not require (a high level of precision or a range of related values)..." ?

Reply:
the first one on top :)